packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -99,3 +99,96 @@ export const createCustomFocusIndicatorStyle = (
   },
   [`${KEYBOARD_NAV_SELECTOR} :${options.selector || defaultOptions.selector}`]: style,
 });
+
+/**
+ * @param pressedBorderColor - color used in case user presses again the field.
+ * @param targetChild - primary element if style is applied to a wrapper.
+ */
+export type BottomFocusIndicatorOptions = CreateFocusIndicatorStyleRuleOptions & {
```

Comment:
General question on this approach -- I get that it allows centralization of the focus style, but do we want this baked into `react-tabster`? Unlike the existing `getFocusOutlineStyles`, the bottom focus indicator is extremely theme-specific, even with the passed-in options.

I'm not completely against it, but this does feel like a fairly weird place to put a bunch of very specific, opinionated styling.

Reply:
I first chose this `react-tabster` because `createCustomFocusOutlineStyle`, `getFocusOutlineStyles`, and `createCustomFocusIndicatorStyle` live in it, but I added this to the Wednesday's tech sync agenda to have a discussion on where we should add it. For now, it made sense to keep it in `useFocusIndicatorStyle.ts`, but hopefully we'll find a better place.

Reply:
For reference: On further discussion with the team, we've decided to leave it out of this PR and track it in a separate [issue](https://github.com/microsoft/fluentui/issues/22231).